src/antispoof.c

```diff
@@ -162,10 +162,21 @@ static struct pg_brick_config *antispoof_config_new(const char *name,
 }
 
 static inline int antispoof_arp(struct pg_antispoof_state *state,
-				struct pg_antispoof_arp *a)
+				struct ether_hdr *eth)
 {
 	uint16_t size = state->arps_size;
 	struct pg_antispoof_arp *p;
+	uint16_t etype = eth->ether_type;
+	struct pg_antispoof_arp *a = (struct pg_antispoof_arp *)(eth + 1);
+
+	switch (etype) {
```

Comment:
tbh I find an if/else more readable here:

```
if (unlikely(etype == PG_BE_ETHER_TYPE_RARP))
return -1;
else if  (etype != PG_BE_ETHER_TYPE_ARP)
return 0;
```

you can even make it in 2 line"
```
if (etype != PG_BE_ETHER_TYPE_ARP)
return (etype == PG_BE_ETHER_TYPE_RARP) * -1;
```